packages/react-scripts/backpack-addons/useTsChecker.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const ForkTsCheckerWebpackPlugin =
+  process.env.TSC_COMPILE_ON_ERROR === 'true'
+    ? require('react-dev-utils/ForkTsCheckerWarningWebpackPlugin')
+    : require('react-dev-utils/ForkTsCheckerWebpackPlugin');
+
+const paths = require('../config/paths');
+const appPackageJson = require(paths.appPackageJson);
+const resolve = require('resolve');
+const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
+const skipProductionTypeCheck =
+  bpkReactScriptsConfig.skipProductionTypeCheck || false;
+
+const useTsChecker = (
```

Comment:
Using TS Checker plugin adds around 30% to build time as early tests show. tsc can do the same job better and faster. I make this opt-in for consumers and keep type checking during development

Reply:
Used [this plugin](https://github.com/stephencookdev/speed-measure-webpack-plugin) to measure loaders and plugins speed